addons/l10n_pe/models/account_chart_template.py

```diff
@@ -0,0 +1,17 @@
+from odoo import models
+
+
+class AccountChartTemplate(models.Model):
+
+    _inherit = 'account.chart.template'
+
+    def _prepare_all_journals(self, acc_template_ref, company, journals_dict=None):
+        """ If Peruvian chart, we don't create sales journal as we need more data to create it properly """
+        journals = super()._prepare_all_journals(acc_template_ref, company, journals_dict=journals_dict)
+        if company.country_id == self.env.ref('base.pe'):
+            for journal in journals:
+                if journal['type'] == 'sale':
+                    journal.update({'l10n_latam_use_documents': True,
+                                    'default_debit_account_id': self.env.ref('l10n_pe.chart1213').id,
+                                    'default_credit_account_id': self.env.ref('l10n_pe.chart1213').id})
```

Comment:
Move to edi module too somehow?

Reply:
Removed this method, neither the account was necessary and the field will be set in the edi!